### ISSUE TITLE
Consider keeping a changelog and publishing git tags for releases

### ISSUE THREAD

**Issue by shepmaster** [rdp/os#45]
1.1.1 appears to have been released, but it's difficult to see exactly what changed without either of these artifacts. It *appears* to be [this diff](https://github.com/rdp/os/compare/a9c1af3488350719986e234ef3e45d1e303f00b8...4ea13b0ce2a27fe21ef4371201bb4fbf277e9410).

**Comment by rdp** [comment]
good idea, I need to stop being lazy, see also https://github.com/rdp/os/pull/47
